packages/protocol/contracts/common/Proxy.sol

```diff
@@ -138,4 +147,20 @@ contract Proxy {
     }
     emit OwnerSet(newOwner);
   }
+
+
+  /**
+   * @dev _isContract detect whether the address is 
+   *      a contract address or externally owned account (EOA)
+   * WARNING: Calling this function from a constructor will return 0
```

Comment:
Nit: function returns `bool`, so I guess to be precise this should say `false`, not `0`?

Reply:
:+1: